From 448179a36e2d4bb74efbbf549d73b5a262943df4 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Wed, 6 May 2015 15:19:16 +0200
Subject: [PATCH 04/16] trace-cmd: update to 2.4.2

fixes compile error:
(observed w. target arm64, all kernel build opts enabled)
error: call to '__open_missing_mode' declared with attribute
error: open with O_CREAT in second argument needs 3 arguments

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 package/devel/trace-cmd/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/package/devel/trace-cmd/Makefile b/package/devel/trace-cmd/Makefile
index 49b5f10..6ed6673 100644
--- a/package/devel/trace-cmd/Makefile
+++ b/package/devel/trace-cmd/Makefile
@@ -1,13 +1,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=trace-cmd
-PKG_VERSION:=v2.4
+PKG_VERSION:=v2.4.2
 PKG_RELEASE=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=git://git.kernel.org/pub/scm/linux/kernel/git/rostedt/trace-cmd.git
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
-PKG_SOURCE_VERSION:=0517388225e776a865fa07fa2ce105e97bd19212
+PKG_SOURCE_VERSION:=01f8e669cb035aa911f1ee0e3f94d535cbcca78f
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 # PKG_MIRROR_MD5SUM:=6a08dfa2519a969d184e7bb5bb390620
 PKG_INSTALL:=1
-- 
2.4.0

